### ISSUE TITLE
PointCloudLayer fp64 + COORDINATE_SYSTEM.LNGLAT mode is not working

### ISSUE THREAD

**Issue by cmsdkfz** [uber/deck.gl#903]
I just created a simple point cloud layer as below, if I turn off 64fp, it renders OK, but if I set 64fp: true, nothing is rendered.

function generateRandomPoints() {
  var points = [];
  for (var i = 0; i < 10000; i++) {
    var diff1 = Math.random() / 1000;
    var diff2 = Math.random() / 1000;
    points.push({
      position: [-122.41669 + diff1, 37.7853 + diff2, 0],
      normal: [0, 0, 1]
    });
  }
  return points;
}

function generatePointLayer(points) {
  return new PointCloudLayer({
    id: 'point-cloud-layer',
    data: points,
    projectionMode: COORDINATE_SYSTEM.LNGLAT,
    getPosition: d => d.position,
    getNormal: d => d.normal,
    radiusPixels: 2,
    fp64: true
  });
}

**Comment by Pessimistress** [comment]
@cmsdkfz Can you share more information about your browser, OS and GPU?

**Comment by cmsdkfz** [comment]
Chrome Version 60.0.3112.101 (Official Build) (64-bit)
macOS sierra
AMD Radeon R9 M370X 2048 MB
Intel Iris Pro 1536 MB

**Comment by 1chandu** [comment]
@cmsdkfz, @ibgreen @Pessimistress I was able to reproduce the problem, working on the fix.

For some reason happens only when coordinate system is `LNGLAT`.

**Comment by ibgreen** [comment]
`fp64` is only needed and used in `LNGLAT` mode. For `METER_OFFSETS` 32-bit precision is sufficient.

**Comment by 1chandu** [comment]
The 64 bit version for vertex shaders needs fixes.
Removing usage of unset uniform 'screenSize`, partially fixed the problem, points get rendered but their position is off by a small offset and size of the point do no match when compared non fp64 rendering, needs more digging.

**Comment by ibgreen** [comment]
There might be two things going on here:
1) the original breakage of this layer
2) my refactor of the project shader module in a recent PR.

The intention was that the refactored `project` module would still set all the old uniforms for backwards compatibility. Maybe I forgot `screenSize`, we could add it back.

I also added a `project_pixel` function to the project module, and used it in the 32 bit shader.
* Can it also be used in the 64 bit?
* Is it giving incorrect results in 32 bits?

I am pretty sure the report on the PointCloudLayer being broken came before I landed my PRs. I could be worth going back to a commit before those and try again.
